framework/Sources/VGSFramework/Core/VGSCollect+extension.swift

```diff
@@ -7,7 +7,9 @@
 //
 
 import Foundation
+#if canImport(UIKit)
```

Comment:
What happened, if someone can't import `UIKit`?

Reply:
UIKit is default iOS UI module, so this can happened only if somebody decide to run our SDK not on iOS(like on MacOS, etc).  In that case SDK will not run at all.  Since we use Swift Package Manager which allow to distribute sdk on various Apple platforms, it's required to add this check.